site_preference_builder/ref/system-objecttype-extensions.xml

```diff
@@ -14,57 +14,13 @@
                 <externally-managed-flag>false</externally-managed-flag>
                 <min-length>0</min-length>
             </attribute-definition>
-            <attribute-definition attribute-id="almaRefundedAmount">
```

Comment:
I'am not sure to understand this removing.
Could we speak about that when you will work on this PR please ?

Reply:
As we want to be able to toggle this feature, this file wont carry this property anymore.